Change list type from unordered to ordered (#1383)
temporalio/documentation: docs/concepts/what-is-a-data-converter.md

> To use a custom Data Converter, provide it to:

Reviewer: ```suggestion
To use a custom Data Converter, provide it in the following contexts:
```
Per https://developers.google.com/style/lists#introductory-sentences-for-lists.

- The Client and Worker in the SDKs you use.
- Temporal Web via [`tctl data-converter web`](/tctl/dataconverter/web) (for displaying decoded data in the Web UI).
- `tctl` via [`--data-converter-plugin`](/tctl/#--data-converter-plugin) (for displaying decoded headers in `tctl` output).

Custom Data Converters are not applied to all data:

- `searchAttributes` are always encoded with JSON.

> - Headers are not encoded by the SDK (the one exception will be—once implemented—the SDK [running OTel baggage through custom Codecs](https://github.com/temporalio/sdk-typescript/issues/514)).

Reviewer: ```suggestion
- Headers are not encoded by the SDK (the one exception will be—when implemented—the SDK [running OTel baggage through custom Codecs](https://github.com/temporalio/sdk-typescript/issues/514)).
```

> Doing encryption in a custom Data Converter ensures that all application data is encrypted while:

Reviewer: ```suggestion
Doing encryption in a custom Data Converter ensures that all application data is encrypted during the following actions:
```

- Being sent to/from Temporal Server.
- Moving inside Temporal Server.
- Stored by Temporal Server.

> Then data only exists unencrypted in memory on the Client and in the Worker Process that is executing Workflows and Activities on hosts that the application developer controls.

Reviewer: ```suggestion
Then data exists unencrypted in memory only on the Client and in the Worker Process that is executing Workflows and Activities on hosts that the application developer controls.
```
